pkg/utils/chart/chart.go

```diff
@@ -77,6 +84,20 @@ func (c *Chart) Apply(
 	if err != nil {
 		return errors.Wrapf(err, "could not apply chart '%s' in namespace '%s'", c.Name, namespace)
 	}
+
+	// Delete disabled subcharts
+	for _, sc := range c.SubCharts {
+		v, err := evaluateCondition(additionalValues, sc.Condition)
```

Comment:
Can you add unit tests about the new handling that is being introduced with this PR?

Reply:
There are no unit tests at all for `chart.Apply` right now, but ok, I will add them.

Reply:
Done